views/signal_annotate.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2022-2025 Searis AS
```

Comment:
In a new file, you don't need to date the copy-right back to 2022
```suggestion
// Copyright 2025 Searis AS
```

Reply:
I didn't know that! :) Thanks. I have updated the commit.